sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewViewModel.groovy

```diff
@@ -22,6 +22,8 @@ class ProjectOverviewViewModel {
 
     @Bindable ProjectSummary selectedProject
     @Bindable String generatedManifest
+    //ToDo this has to be set by the ResourceService
+    @Bindable Boolean projectIsSubscribed
```

Comment:
Only add the variable if its filled/used. If you think something is missing feel fee to open an issue for that

```suggestion

```